activiti-cloud-services-query/activiti-cloud-services-query-model/src/main/java/org/activiti/cloud/services/query/model/TaskEntity.java

```diff
@@ -367,6 +369,20 @@ public void setTaskCandidateUsers(Set<TaskCandidateUser> taskCandidateUsers) {
         this.taskCandidateUsers = taskCandidateUsers;
     }
 
+    public Set<String> getCandidateUsers(){
```

Comment:
Same question as for the PR in Activiti core. Any reason to use `Set` instead of `List` here.

Reply:
The answer is the same :) I just preferred Set instead of List in order to be sure that values aren't duplicated.